### PR TITLE
Migrate models/aragon/*.sql and models/dao/addresses/*.sql

### PR DESCRIPTION
**Pull request by Hosuke**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by henrystats** [comment]
hola ser @Hosuke 👋 - I was supposed to work on this, this week... 

I built the Aragon/dao & dao section (balances, addresses & transactions) whilst working with the Aragon DAO/team.. 

Migrating the spells this week and I just came across your pr... It was actually a bounty set up with them to migrate all the underlying spells.

**Comment by henrystats** [comment]
was going to building models for the other tools like syndicate, daohaus etc

**Comment by Hosuke** [comment]
> hola ser @Hosuke 👋 - I was supposed to work on this, this week...
> 
> I built the Aragon/dao & dao section (balances, addresses & transactions) whilst working with the Aragon DAO/team..
> 
> Migrating the spells this week and I just came across your pr... It was actually a bounty set up with them to migrate all the underlying spells.

I actually migrated most of dao sectors in my local branches last week. You can recycle my work if you need any.

File: tests/generic/dao_transactions_creator_tool.sql [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {% test dao_transactions_creator_tool(model, column_name, dao_transactions_seed) %}
 
+    -- TODO: Modify the test to dune sql compatible @henrystats
```
Comment (Hosuke) [inline tests/generic/dao_transactions_creator_tool.sql:3]: Here is the generic test that we need to migrate as well.

**Review by Hosuke — COMMENTED** [review]
@henrystats 
I have synced all my translated dao spells in this PR.
You can pull from my dev branch.
I will pass my work to you.

**Comment by henrystats** [comment]
Thank you very much ser @Hosuke, will set up pr later today.

**Pull request closed without merging**